### PR TITLE
Tell users about the new beta language support

### PR DESCRIPTION
**Pull request by LkeMitchll**
screen shot 2017-01-30 at 11 40 34 am

Languages in 'beta' are:

- Python
- Elixir
- TypeScript

relates to #1305

### CODE REVIEW

File: app/assets/images/languages/elixir.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="100" height="100" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><title>Group 2</title><g fill="none" fill-rule="evenodd"><circle cx="50" cy="50" r="50" fill-rule="nonzero" fill="#4E2A8E"/><path d="M31.099 55.337A18.947 18.947 0 0 0 31 57.27C31 67.615 39.385 76 49.73 76c10.343 0 18.728-8.385 18.728-18.73a18.67 18.67 0 0 0-5.413-13.17C48.298 27.348 51.038 18 51.038 18s-18.522 9.857-19.94 37.337z" stroke="#FFF" stroke-width="4" stroke-linecap="round" stroke-linejoin="round"/></g></svg>
```
Comment (teoljungberg) [inline app/assets/images/languages/elixir.svg]: This lacks a proper newline.

File: app/assets/images/languages/python.svg [hunk 2]
```diff
@@ -1,36 +1 @@
...
....639 22.5)"><mask id="e" fill="#fff"><use xlink:href="#d"/></mask><path fill="#FFF" fill-rule="nonzero" mask="url(#e)" d="M-17.61-30.35h104.5V82.416h-104.5z"/></g><path d="M14.445 43.444c14.597 0 13.685-6.33 13.685-6.33l-.016-6.558h-13.93v-1.97h19.463s9.34 1.06 9.34-13.668C42.987.188 34.835.712 34.835.712H29.97v6.834s.26 8.153-8.023 8.153H8.13s-7.76-.127-7.76 7.5v12.612s-1.18 7.632 14.075 7.632v.001zm7.68-4.41a2.503 2.503 0 0 1-2.505-2.506 2.503 2.503 0 0 1 2.506-2.506 2.504 2.504 0 0 1 2.506 2.506 2.503 2.503 0 0 1-2.506 2.506h-.001z" fill-rule="nonzero" fill="url(#f)" transform="translate(35.639 36.5)"/><path d="M50.084 79.944c14.597 0 13.685-6.33 13.685-6.33l-.016-6.558h-13.93v-1.97h19.463s9.34 1.06 9.34-13.668c0-14.73-8.152-14.206-8.152-14.206h-4.865v6.834s.26 8.153-8.023 8.153H43.769s-7.76-.127-7.76 7.5v12.612s-1.18 7.632 14.075 7.632v.001zm7.68-4.41a2.503 2.503 0 0 1-2.505-2.506 2.503 2.503 0 0 1 2.506-2.506 2.504 2.504 0 0 1 2.506 2.506 2.503 2.503 0 0 1-2.506 2.506h-.001z"/><g transform="translate(35.639 36.5)"><mask id="h" fill="#fff"><use xlink:href="#g"/></mask><path fill="#FFF" fill-rule="nonzero" mask="url(#h)" d="M-32.61-44.35h104.5V68.416h-104.5z"/></g></g></g></svg>
```
Comment (teoljungberg) [inline app/assets/images/languages/python.svg]: Ditto regarding the newline.

File: app/assets/images/languages/typescript.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg width="100" height="100" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg"><title>Group 2 Copy</title><g fill="none" fill-rule="evenodd"><circle fill="#178FCF" fill-rule="nonzero" cx="50" cy="50" r="50"/><g fill="#FFF"><path fill-rule="nonzero" d="M50.7 36.1H39.6v34.3h-4.5V36.1H24V32h26.6v4.1z"/><path d="M54 68.7v-3c2.8 1.8 5.5 2.6 8.4 2.6 3 0 5.3-.6 6.8-1.9 1.6-1.2 2.3-3 2.3-5.2 0-2-.5-3.5-1.6-4.7-1-1.2-3.3-2.8-6.8-4.8-3.9-2.3-6.4-4.2-7.4-5.7s-1.6-3.3-1.6-5.3c0-2.7 1.1-5 3.2-6.9 2.1-1.9 4.9-2.8 8.4-2.8 2.3 0 4.6.4 6.9 1.2V35c-2.3-1-4.7-1.5-7.2-1.5-2.6 0-4.7.7-6.2 2s-2.3 3-2.3 5 .5 3.5 1.6 4.7c1 1.2 3.3 2.7 6.8 4.7 3.6 2 6 3.8 7.2 5.4 1.2 1.6 1.8 3.4 1.8 5.5 0 2.9-1 5.3-3.1 7.2-2 1.9-4.9 2.8-8.7 2.8-1.3 0-2.8-.2-4.6-.6-1.6-.5-2.9-1-3.9-1.5z"/></g></g></svg>
```
Comment (teoljungberg) [inline app/assets/images/languages/typescript.svg]: Ditto regarding the newline.

**Comment by teoljungberg** [comment]
I wonder if we event want to announce these linters, since I feel they haven't been battle-tested enough yet.

@gylaz What do you think?

**Comment by croaky** [comment]
Thoughts on adding just the languages that have been tested enough? What does a user do with the "beta" indicator?

It seems like we can add Elixir to this list without the "beta" indicator based on what I've heard from @jakecraige and others. @jackvnimble?

Are we actively testing Python, too? @georgebrock @mike-burns do you have feedback on whether you consider Hound's Python support tested, whether a "beta" indicator is meaningful, whether you think it should be promoted on the homepage and documented in the config section?

**Comment by mike-burns** [comment]
I have not made use of the Python linter in Hound.

**Comment by jakecraige** [comment]
I'm comfortable with Elixir being publicly announced as beta. It's been doing what it's supposed to on Jack's project without issue for a few weeks at least now.

**Comment by jackvnimble** [comment]
Yeah, it's been solid.

**Comment by jakecraige** [comment]
@jackvnimble Cept those module doc errors MIRITE?!

**Comment by jackvnimble** [comment]
I got rid of them with this one weird trick.

**Comment by georgebrock** [comment]
I've not used the Python linter recently enough to know if it should qualify as beta or not.

**Comment by gylaz** [comment]
I think we could just add Elixir here without "beta". I'm not comfortable mentioning Python or TypeScript, since we aren't actively testing/using/maintaining it.

**Comment by gylaz** [comment]
@LkeMitchll mind updating the icons without "beta"? Or just ping them to me in slack and I can do this.

**Comment by tysongach** [comment]
I rebased this branch and updated it based on feedback above.

screen shot 2017-06-02 at 16 58 59

**Comment by tysongach** [comment]
[Update “Currently style checking” on home page](https://trello.com/c/Xxfg1cQH/817-update-%E2%80%9Ccurrently-style-checking%E2%80%9D-on-home-page)

**Pull request merged**